---
title: C28163
ms.date: 11/04/2016
ms.topic: reference
f1_keywords:
- C28163
helpviewer_keywords:
- C28163
ms.assetid: 24fecbde-1c96-4a45-82f7-9f47cfc0ef11
author: corob-msft
ms.author: corob
manager: markl
ms.workload:
- multiple
ms.openlocfilehash: 87c5a405a47bd35e82d971b078141fbad4c4b250
ms.sourcegitcommit: 68f893f6e472df46f323db34a13a7034dccad25a
ms.translationtype: MT
ms.contentlocale: zh-CN
ms.lasthandoff: 02/15/2020
ms.locfileid: "77270053"
---
# <a name="c28163"></a>C28163
警告 C28163：绝不应从 try/except 块中调用函数

 当函数的类型不应包含在 `try/except` 块中的 `try/except` 块中时，将报告此警告。 代码分析工具至少发现了一个路径，其中，所调用的函数都位于 `try/except` 块中。
